src/Microsoft.VisualBasic.Core/ref/Microsoft.VisualBasic.Core.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AssemblyVersion>10.0.4.0</AssemblyVersion>
```

Comment:
I believe we can omit this since it is defined here: https://github.com/333fred/corefx/blob/983c0b70807ed4081360fc3cedad3117a58d8454/src/Microsoft.VisualBasic.Core/Directory.Build.props#L5